### DISCUSSION TITLE
Termux Hangs in Processing

### DISCUSSION THREAD

**Discussion by NimaDolatabadi** [termux/termux-packages#10986] [General]
I have a software completely compiled and defined inside bashrc
This software is a command line based and when i'm running it feels kinda stuck,
It has to go to the next line and show:
SAC>
and there i can type my commands.
But it feels kinda stuck..
I hope you can help me!
Regards.
Nima

![Screenshot_20220615-130618.png](https://user-images.githubusercontent.com/32169409/173783304-b9b5b823-144b-4243-aade-5758f37f5cd3.png)

**Comment by Grimler91** [comment]
Most likely a problem with the program you are running. 

Since we don't have access to the program and you haven't linked to the source code or told us how you configured and compiled it, there's not really anything for us to do here

**Reply by NimaDolatabadi** [reply]
> Most likely a problem with the program you are running. 
> 
> Since we don't have access to the program and you haven't linked to the source code or told us how you configured and compiled it, there's not really anything for us to do here


It was kinda tricky to do so
I'm sharing the link to download the source code:
https://www.mediafire.com/download/z5gul7haa9p8acv

I've installed the followings to complete the compilation,
Pkg install binutils
Pkg install libx11
Pkg install x11-repo
Pkg install xorgproto
Pkg install clang,
Libncurses

And there is a problem from config.guess so i copied that from **/usr/share/Automake/config.guess** to **/sac-102.0/config**
And the compilation is done by defining the prefix in ../SAC just in case to be tidy :-)

The process is like:
./configure
Make
Make install

And made a simbolic link of bash.bashrc inside home as .bashrc (to access better)

Then defined the path inside bashrc as following:
source /data/data/com.termux/files/home/src/SAC/bin/sacinit.sh

This is based on my directories.

I hope it could help you do so,
Regards
Nima
